pkg/apis/garden/v1beta1/types.go

```diff
@@ -113,6 +113,9 @@ type MachineImage struct {
 	Name string `json:"name"`
 	// Version is the version of the image.
 	Version string `json:"version"`
+	// ForceUpdateTime defines when a shoot running this image version will be forcefully updated to the latest cloud profile specified in the respective cloud profile
+	// +optional
+	ForceUpdateTime *string `json:"forceUpdateTime,omitempty"`
```

Comment:
I suppose you've chosen a string type to allow timezones to be configured. But since a machine image is handled globally for all Shoot clusters of the same cloud provider, I don't see any advantage of having a timezone.
In this case I'd rather go with [metav1.Time](https://github.com/kubernetes/apimachinery/blob/391ed67afa7b//pkg/apis/meta/v1/time.go#L33).

Any thoughts @adracus @zanetworker?

Reply:
Furthermore, didn't we agree to call it `ExpirationDate`?

Imho, `ForceUpdateTime` means that you forcefully want to upgrade to that `Version`. But what you actually do is to check whether the time in `ForceUpdateTime` has been passed and jump to a latter version.